yarn-error.log

```diff
@@ -1,5 +1,5 @@
 Arguments: 
-  /Users/paltman/.nvm/versions/node/v10.16.3/bin/node /usr/local/Cellar/yarn/1.22.4/libexec/bin/yarn.js install
+  /Users/paltman/.nvm/versions/node/v10.16.3/bin/node /usr/local/Cellar/yarn/1.22.4/libexec/bin/yarn.js add --dev -W @apollo/cache-inmemory
```

Comment:
@paltmanI didn't realize `yarn-error.log` was being tracked; let's add it to .gitignore